### PR TITLE
docs: mention "london" among accepted values for "evmVersion"

### PR DESCRIPTION
**Pull request by paulrberg**
- [x] This PR includes a **documentation change**, so its branch was created from the `website` branch, and this PR uses the `website` branch as its base branch.

---

Since [v2.6.0](https://github.com/nomiclabs/hardhat/releases/tag/hardhat-core-v2.6.0), Hardhat accepts "london" as one of the allowed values for `evmVersion`.

### CODE REVIEW

**Comment by fvictorio** [comment]
I wonder if listing the hardforks make sense here. The actual list depends on the solc version you are using, right? Maybe we shouldn't list any hardfork at all and just mention that is a solc configuration.

@alcuadrado @feuGeneA what do you think?

**Comment by feuGeneA** [comment]
> I wonder if listing the hardforks make sense here. The actual list depends on the solc version you are using, right? Maybe we shouldn't list any hardfork at all and just mention that is a solc configuration.
> 
> @alcuadrado @feuGeneA what do you think?

I think it makes sense to list them here, but I suggest we (a) prefix the sentence with "For example, ..." rather than just "One of...", and (b) shorten the list to just a few, perhaps just "istanbul, berlin or london" (also note (c) the shift from "and" to "or"), all in order to avoid sounding too authoritative, reinforcing the deference to the solc reference that we're already linking to previously in this doc.

Also, @paulrberg note that you introduced a line break into the text; I believe that's the reason for the failed linter run in the PR.

**Comment by alcuadrado** [comment]
I agree with @feuGeneA here!

File: docs/guides/compile-contracts.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ module.exports = {
 
 - `optimizer`: an object with `enabled` and `runs` keys. Default value: `{ enabled: false, runs: 200 }`.
 
-- `evmVersion`: a string controlling the target evm version. One of `homestead`, `tangerineWhistle`, `spuriousDragon`, `byzantium`, `constantinople`, `petersburg`, `istanbul`, and `berlin`. Default value: managed by `solc`.
+- `evmVersion`: a string controlling the target evm version. One of `homestead`, `tangerineWhistle`, `spuriousDragon`, `byzantium`, `constantinople`, `petersburg`, `istanbul`, `berlin` or `london`. Default value: managed by `solc`.
```
Comment (feuGeneA) [inline docs/guides/compile-contracts.md:62]:
```suggestion
- `evmVersion`: a string controlling the target evm version. For example: `istanbul`, `berlin` or `london`. Default value: managed by `solc`.
```

**Review by fvictorio — APPROVED** [review]

**Pull request merged**